src/frontend-samples/query-volume-app/VolumeQueryUI.tsx

```diff
@@ -0,0 +1,287 @@
+          <span>Clip Volume</span>
+          <Toggle
+            isOn={this.state.isClipVolumeOn}
+            onChange={this._onToggleClipping}
+          />
+        </div>
+        <hr></hr>
+        <div className="sample-options-3col">
+          <span>Coloring:</span>
+          <Button buttonType={ButtonType.Blue} disabled={!this.state.isVolumeBoxOn} onClick={this._onClickApplyColoring}>Apply</Button>
+          <Button buttonType={ButtonType.Blue} onClick={this._onClickClearColoring}>Clear</Button>
+        </div>
+        <div className="sample-options-3col">
+          <div style={{ display: "flex", alignItems: "center", marginLeft: "20px", marginRight: "20px" }}>
+            <span>{ElementPosition.InsideTheBox}</span>
+            <ColorPickerButton style={{ marginLeft: "10px" }}
+              initialColor={this.state.spatialElementsColors.insideColor}
+              onColorPick={(color: ColorDef) => this._onColorPick(color, ElementPosition.InsideTheBox)}
+            />
+          </div>
+          <div style={{ display: "flex", alignItems: "center" }}>
+            <span>{ElementPosition.OutsideTheBox}</span>
+            <ColorPickerButton style={{ marginLeft: "10px" }}
+              initialColor={this.state.spatialElementsColors.outsideColor}
+              onColorPick={(color: ColorDef) => this._onColorPick(color, ElementPosition.OutsideTheBox)}
+            />
+          </div>
+          <div style={{ display: "flex", alignItems: "center" }}>
+            <span>{ElementPosition.Overlap}</span>
+            <ColorPickerButton style={{ marginLeft: "10px" }}
+              initialColor={this.state.spatialElementsColors.overlapColor}
+              onColorPick={(color: ColorDef) => this._onColorPick(color, ElementPosition.Overlap)}
+            />
+          </div>
+        </div>
+        <hr></hr>
+        <div className="sample-options-2col">
+          <span style={{ whiteSpace: "nowrap" }}>List Colored Elements:</span>
+          <select onChange={this._onSelectionListElements.bind(this)} value={this.state.elementsToShow.position}>
+            <option value={ElementPosition.InsideTheBox}> {ElementPosition.InsideTheBox} </option>
+            <option value={ElementPosition.OutsideTheBox}> {ElementPosition.OutsideTheBox} </option>
+            <option value={ElementPosition.Overlap}> {ElementPosition.Overlap} </option>
+          </select>
+        </div>
+        <div className="table-wrapper">
+          <select multiple style={{ maxHeight: "100px", overflowY: "scroll", overflowX: "hidden", whiteSpace: "nowrap" }}>
+            {this.state.elementsToShow.elements.slice(99).map((element) => <option key={element.id} style={{ listStyleType: "none", textAlign: "left" }}>{element.name}</option>)}
+          </select>
+        </div>
+        <span style={{ color: "grey" }} className="table-caption">Selected {this.state.elementsToShow.elements.length} elements, showing top 100.</span>
```

Comment:
These elements are not selected.  Please change the message to:  "List contains <count> elements, showing first 100."